web/frontend/src/components/base.vue

```diff
@@ -50,6 +52,11 @@ const power = $ref(50);
 const pressed = new Set<Keys>();
 let stopped = true;
 
+const keyboardStates = {
+  tempDisable: ref(false),
```

Comment:
using `$ref` here didn't work so I stuck with `ref`

Reply:
I'd recommend only using refs top level, and not inside objects. They weren't intended for use in objects, and you can alternatively set objects as refs. That'll also fix your problem of not being able to use `$ref`